target/linux/mediatek/filogic/base-files/etc/hotplug.d/firmware/11-mt76-caldata

```diff
@@ -16,6 +16,9 @@ case "$FIRMWARE" in
 	;;
 "mediatek/mt7981_eeprom_mt7976_dbdc.bin")
 	case "$board" in
+	comfast,cf-e393ax)
+		caldata_extract "Factory" 0x0 0x1000
```

Comment:
It should be enough to reference the EEPROM in device tree. No need to do it again in userspace.

Reply:
Thanks, could you give an example, im not up to speed with these newer changes in the dts?

Reply:
So im not sure if this is the approach for this?
```

factory: partition@180000 {
				label = "Factory";
				reg = <0x180000 0x0200000>;
				read-only;

				compatible = "nvmem-cells";
				#address-cells = <1>;
				#size-cells = <1>;

				macaddr_factory_e000: macaddr@e000 {
					reg = <0xe000 0x6>;
				};

				macaddr_factory_8000: macaddr@8000 {
					reg = <0x8000 0x0>;
				};
			};
```
```
&wifi {
	status = "okay";
	mediatek,mtd-eeprom = <&factory 0x0>;
	nvmem-cells = <&macaddr_factory_8000>;
	nvmem-cell-names = "mac-address";
};
```
For the 11-mt76-caldata and the mt7981_eeprom_mt7976_dbdc.bin how would that be created? or is it still needed?